docker/Makefile

```diff
@@ -61,8 +66,23 @@ dcmqi.generate_package: prereq.pull_dockcross
 	$(TMP)/dockcross ninja -C$(BUILD_DIR)/dcmqi-build package
 
 # Build "dcmqi" docker image
+# 
+# Docker expects package content to be in directory dcmqi-linux. As a
+# workaround, the content of the cpack-generated package, stored in
+# dcmqi-<version>-linux-<date>-<hash> directory is copied to dcmqi-linux.
+#
 dcmqi.docker_image:
-	tar -xzvf $(ROOT_DIR)/$(BUILD_DIR)/dcmqi-build/dcmqi-linux.tar.gz -C $(DOCKER_DIR)/dcmqi
+	$(info $$ROOT_DIR ${ROOT_DIR})
+	$(info $$BUILD_DIR ${BUILD_DIR})
+	$(eval PACKAGE_FILE_NAME := $(shell ls $(ROOT_DIR)/build/dcmqi-build/_CPack_Packages/linux/TGZ/dcmqi-*-linux*.tar.gz))
```

Comment:
@jcfr I defined `SHELL` above, since without it on Ubuntu, `/bin/sh` would be used. It is no longer critical, since I changed the approach of how to get the directory name, but I think it is good to keep `SHELL` definition, since otherwise it is super confusing that otherwise shell behavior can be different across linux distributions.

In this line `shell` is a make function, see https://www.gnu.org/software/make/manual/html_node/Shell-Function.html.